test/playwright.fixtures.ts

```diff
@@ -114,6 +114,7 @@ registerWorkerFixture('defaultBrowserOptions', async ({browserName}, test) => {
 registerWorkerFixture('playwright', async ({browserName}, test) => {
   const {coverage, uninstall} = installCoverageHooks(browserName);
   if (options.WIRE) {
+    require('../lib/utils/utils').setDevMode();
```

Comment:
What is dev mode?

Reply:
Former setUnderTest.

Reply:
Should we add back the checks that playwright wasn't already required here? Seems important again.